velox/exec/Operator.h

```diff
@@ -459,6 +459,17 @@ class Operator : public BaseRuntimeStatWriter {
     return identityProjections_;
   }
 
+  /// Returns a list of filtered projections, which are used for filter pushdown
+  /// to pass through. If the operator has initialized filtered projections,
+  /// such as HashAggregation, return them directly. For most operators, this
+  /// should be forward to identityProjections().
+  const std::vector<IdentityProjection>& filteredProjections() const {
+    if (!filteredProjections_.empty()) {
```

Comment:
Let's remove this field, make this API virtual and have default implementation that returns identityProjections.